packages/edit-site/src/components/global-styles/font-library-modal/context.js

```diff
@@ -60,12 +61,18 @@ function FontLibraryProvider( { children } ) {
 		records: libraryPosts = [],
 		isResolving: isResolvingLibrary,
 		hasResolved: hasResolvedLibrary,
-	} = useEntityRecords( 'postType', 'wp_font_family', { refreshKey } );
+	} = useEntityRecords( 'postType', 'wp_font_family', {
+		refreshKey,
+		_embed: true,
+	} );
 
 	const libraryFonts =
-		( libraryPosts || [] ).map( ( post ) =>
-			JSON.parse( post.content.raw )
-		) || [];
+		( libraryPosts || [] ).map( ( post ) => {
+			post.font_family_settings.fontFace = post._embedded.font_faces.map(
```

Comment:
On a WP instance where I have a mix of the old Font Library structure and the new, I'm seeing this error: `context.js:71 Uncaught TypeError: Cannot read properties of undefined (reading 'font_faces')`, so I think we should check for `font_faces` here:

```suggestion
			post.font_family_settings.fontFace = post._embedded?.font_faces.map(
```

Reply:
Oh, I did not test with sites that have both. We should probably make this work with existing font library data structure also in this PR...?

Reply:
Scratch that — we decided to add functionality to auto-create wp_font_face posts in the server side for the previous version of wp_font_family, followed by implementing the UI to handle mismatches between installed vs activated fonts.